app/sidekiq/host_content_update_job.rb

```diff
@@ -0,0 +1,18 @@
+class HostContentUpdateJob < DependencyResolutionJob
+private
+
+  def downstream_live(dependent_content_id, locale)
+    return if draft?
+
+    DownstreamLiveJob.perform_async_in_queue(
+      DownstreamLiveJob::LOW_QUEUE,
+      "content_id" => dependent_content_id,
+      "locale" => locale,
+      "message_queue_event_type" => "host_content",
```

Comment:
**question/suggestion**: given this is doing the same as `DependencyResolutionJob`'s `downstream_live` method but passing in a different `message_queue_event_type`, could it be an idea to extract that to a shared method with an argument for `message_queue_event_type`? Or can you see this behaviour diverging in future updates?

This would prevent changes to `DependencyResolutionJob`'s behaviour being missed when updating host content, but this may be by design!